protocol/messenger_mailserver.go

```diff
@@ -379,6 +379,39 @@ func (m *Messenger) RequestAllHistoricMessages(forceFetchingBackup, withRetries
 	return allResponses, nil
 }
 
+const missingMessageCheckPeriod = 30 * time.Second
+
+func (m *Messenger) checkForMissingMessages() {
+	go func() {
+		t := time.NewTimer(missingMessageCheckPeriod)
```

Comment:
I am wondering why not ticker? It would simplify the code and avoid potential pitfalls with resetting timer.

Reply:
This is actually an error. It should have been `t := time.NewTimer(0)` so it ticks immediatly.

Reply:
Otherwise i'd have to wait `missingMessageCheckPeriod` seconds for the ticker to tick